app/helpers/remote_execution_helper.rb

```diff
@@ -122,6 +122,11 @@ def job_invocation_task_buttons(task)
                          :title => _('Try to cancel the job'),
                          :disabled => !task.cancellable?,
                          :method => :post)
+      buttons << link_to(_('Abort Job'), abort_foreman_tasks_task_path(task),
+                         :class => 'btn btn-danger',
+                         :title => _('Try to abort the job'),
```

Comment:
The title is not very descriptive, what about `Abort the the job: trying to cancel without waiting for the results from the remote hosts)` or something like that?